processor/spanintentprocessor/factory.go

```diff
@@ -0,0 +1,56 @@
+package spanintentprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor"
```

Comment:
Can we ensure the code coverage? E.g.: I see factory.go, but there's no test file.